packages/inference/test/HfInference.spec.ts

```diff
@@ -645,11 +639,114 @@ describe.concurrent(
 		});
 
 		it("endpoint - makes request to specified endpoint", async () => {
-			const ep = hf.endpoint("https://api-inference.huggingface.co/models/google/flan-t5-xxl");
+			const ep = hf.endpoint("https://api-inference.huggingface.co/models/openai-community/gpt2");
 			const { generated_text } = await ep.textGeneration({
 				inputs: "one plus two equals",
 			});
-			expect(generated_text).toEqual("three");
+			assert.include(generated_text, "three");
+		});
+
+		it("chatCompletion modelId - OpenAI Specs", async () => {
+			const res = await hf.chatCompletion({
+				model: "mistralai/Mistral-7B-Instruct-v0.2",
+				messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+				max_tokens: 500,
+				return_full_text: false,
+				temperature: 0.1,
+				seed: 0,
+			});
+			if (res.choices && res.choices.length > 0) {
+				const completion = res.choices[0].message?.content;
+				expect(completion).toContain("to two");
+			}
+		});
+
+		it("chatCompletionStream modelId - OpenAI Specs", async () => {
+			const stream = hf.chatCompletionStream({
+				model: "mistralai/Mistral-7B-Instruct-v0.2",
```

Comment:
two new tests that have to work, we're assuming that if you're using `chatCompletion` it has to be `/v1/chat/completions` ,